cdci_data_analysis/analysis/email_helper.py

```diff
@@ -268,6 +270,30 @@ def send_job_email(
                                              'attachment',
                                              filename="api_code.py")
 
+    status_details_message = None
+    status_details_title = status
+    if status_details is not None and status_details['status'] != 'successful':
+        if status_details['status'] == 'empty_product':
+            status_details_message = '''Unfortunately, after a quick automated assessment of the request, it has been found that it contains an <b>empty, product</b>.
+To the best of our knowledge, no unexpected errors occurred during processing,
+and if this is not what you expected, you probably need to modify the request parameters. We are sorry.<br>'''
+            status_details_title = 'finished: with empty product'
+        elif status_details['status'] == 'empty_result':
+            status_details_message = '''Unfortunately, after a quick automated assessment of the request, it has been found that it contains an <b>empty, result</b>.
```

Comment:
for user, there is no way to understand what is the difference between empty product and empty result.

Reply:
yes. The meaning is the same.